test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -113,8 +117,12 @@ User ppx flags should appear in merlin config
   Bar: _build/default/bar
   ((STDLIB /MELC_STDLIB/melange)
    (EXCLUDE_QUERY_DIR)
+   (B
+    /MELC_STDLIB/__private__/melange_mini_stdlib/melange/.public_cmi_melange)
```

Comment:
Is it expected that a public cmi path is inside a folder named `__private__`?

Reply:
Yes, it’s a private library